### PR TITLE
I3DBEEZ9 New board

### PR DESCRIPTION
**Pull request by I3DBeeTech**

### CODE REVIEW

**Comment by JamesH1978** [comment]
Thank you for submitting a PR, pleas refer to point 3 in "What to expect in a review" in https://github.com/Klipper3d/klipper/blob/master/docs/CONTRIBUTING.md and provide a signed off by line. And please could you add the line for your config in printers.test in alphabetical order as per point 4 in the guidelines in https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md

Thanks
James

**Comment by I3DBeeTech** [comment]
> Thank you for submitting a PR, pleas refer to point 3 in "What to expect in a review" in https://github.com/Klipper3d/klipper/blob/master/docs/CONTRIBUTING.md and provide a signed off by line. And please could you add the line for your config in printers.test in alphabetical order as per point 4 in the guidelines in https://github.com/Klipper3d/klipper/blob/master/docs/Example_Configs.md
> 
> Thanks James

Thank you... james
We have updated accordingly.

**Comment by JamesH1978** [comment]
thankyou for making the changes. I don't suppose you have a link to the pin out for this board so I can check sanity? Although I assume this is a board your company has made themselves, So I would assume this has been tested on the board and is known to work properly?

Thanks
James

**Comment by I3DBeeTech** [comment]
> thankyou for making the changes. I don't suppose you have a link to the pin out for this board so I can check sanity? Although I assume this is a board your company has made themselves, So I would assume this has been tested on the board and is known to work properly?
> 
> Thanks James

Yes our developers tested it @JamesH1978 
We are launching this board by end of this month and editing our web with only our products .
Thank you for your endless efforts.

File: config/generic-I3DBEEZ9.cfg [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+homing_speed: 50
+
+[stepper_z]
+step_pin: PE13
+dir_pin: PC2
+enable_pin: !PC0
+microsteps: 16
+rotation_distance: 8
+endstop_pin: PG8
+position_endstop: 0.5
```
Comment (JamesH1978) [inline config/generic-I3DBEEZ9.cfg:41]: Please set this to 0, position_endstop: 0, as it is a board config, it should have no settings that could confuse

**Review by JamesH1978 — CHANGES_REQUESTED** [review]
I've made one comment in the code. 

Other than that, looks fine.

Thanks
James

**Comment by KevinOConnor** [comment]
Thanks.

-Kevin

**Pull request merged**